tools/local-test/scripts/steps/build-artifacts.sh

```
@@ -1,18 +1,20 @@
 TEMP_DB_PATH=$TMP_PATH/file-db
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)